### ISSUE TITLE
Jasmine Error "doneConsumer" and Jquery Blocked Iframe Error

### ISSUE THREAD

**Issue by etmarch** [Sanjo/meteor-jasmine#151]
Hi, im getting some strange errors when running Velocity with Jasmine for the first time:

This error right when Jasmine starts: 
``` Uncaught SecurityError: Failed to read the 'contentDocument' property from 'HTMLIFrameElement': Blocked a frame with origin "http://localhost:3000" from accessing a frame with origin "http://localhost:57001". Protocols, domains, and ports must match.```

And I am also receiving these errors:  
``` 
Error: Error invoking Method 'jasmine/doneConsumer':
Method not found [404]

Error: Error invoking Method 'jasmine/specDoneConsumer':
Method not found [404]
```

When I run meteor with velocity debug env variable, I noticed this warning message: 
```
[33mWARN [watcher]: [39mPattern "/Users/evanmarchand/Desktop/meteor/mk/insonic/tests/jasmine/client/unit/**/*-{stub,stubs,mock,mocks}.{js,coffee,litcoffee,coffee.md}" does not match any file.
```

Any help is appreciated.

**Comment by etmarch** [comment]
I am also seeing this error many times: 
``` Error: Exception in template helper:
TypeError: undefined is not a function
    at http://localhost:3000/packages/velocity_html-reporter.js?a615bf3410d795106d7a78c6a94379745879f7df:651:43
    at Array.forEach (native)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?0a80a8623e1b40b5df5a05582f288ddd586eaa18:156:11)
    at Object.Template.velocitySummary.helpers.totalTime (http://localhost:3000/packages/velocity_html-reporter.js?a615bf3410d795106d7a78c6a94379745879f7df:649:7)
    at bindDataContext (http://localhost:3000/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:2727:16)
    at Blaze._wrapCatchingExceptions (http://localhost:3000/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:1606:16)
    at Spacebars.call (http://localhost:3000/packages/spacebars.js?7f53771c84a2eafac2b561c9796dda0d8af8e7f5:171:18)
    at Spacebars.mustacheImpl (http://localhost:3000/packages/spacebars.js?7f53771c84a2eafac2b561c9796dda0d8af8e7f5:108:25)
    at Object.Spacebars.mustache (http://localhost:3000/packages/spacebars.js?7f53771c84a2eafac2b561c9796dda0d8af8e7f5:112:39)
    at null._render (http://localhost:3000/packages/velocity_html-reporter.js?a615bf3410d795106d7a78c6a94379745879f7df:218:22)```

**Comment by Sanjo** [comment]
> Uncaught SecurityError: Failed to read the 'contentDocument' property from 'HTMLIFrameElement': Blocked a frame with origin "http://localhost:3000" from accessing a frame with origin "http://localhost:57001". Protocols, domains, and ports must match.

Can you see which script is causing this error? I think this comes from a browser extension or other third party code.

**Comment by Sanjo** [comment]
The doneConsumer error is fixed in 0.12.3. Regarding the other error you need to check your browser extensions (see previous comment).
